ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/CachingCommitStrategy.scala

```diff
@@ -22,7 +21,7 @@ class CachingCommitStrategy[Result](
       entry: DamlKvutils.DamlLogEntry,
       inputState: Map[DamlStateKey, Option[DamlStateValue]],
       outputState: Map[DamlStateKey, DamlStateValue]): Future[Result] = {
-    cache.putAll(outputState.asJava)
+    outputState.foreach { case (key, value) => cache.put(key, value) }
```

Comment:
Isn't this new version doing the same thing in a more verbose way?

Reply:
Yes, it is, however, `com.daml.caching.Cache` does not have a `putAll` hence this change.